src/main/java/uk/gov/hmcts/reform/hmc/data/HearingPartyEntity.java

```diff
@@ -68,4 +70,6 @@ public class HearingPartyEntity {
     @OneToMany(mappedBy = "hearingParty", cascade = CascadeType.PERSIST, orphanRemoval = true)
     private List<ContactDetailsEntity> contactDetails;
 
+    @OneToMany(cascade = CascadeType.PERSIST, orphanRemoval = true)
```

Comment:
Not convinced this is correct, but the `mappedBy = hearingParty` caused issues that stopped AppContext from starting

Reply:
Fixed by @danlysiak in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/50b833069211b0402b67717fae973ed563286907